assessments/serializer.py

```
@@ -1,85 +1,166 @@
+
 
 class ExamAnswersSerializer(BaseSerializer):
+    # Write-only field allows the frontend to send a list of option IDs in the same payload
+    selected_options = serializers.PrimaryKeyRelatedField(
+        queryset=models.QuestionOptions.objects.all(), many=True, write_only=True, required=False
+    )
+
     class Meta(BaseSerializer.Meta):
         model = models.ExamAnswers
-        fields = '__all__'
+        fields = ["id", "exam_submission", "question", "answer_text", "selected_options", "created_date", "status"]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)